code/about.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <menu>
+        <a href=./index.html>Home</a>
+        <a href=./help.html>Das kannst du tun</a>
+        <a href=./newhome.html>Zuhause gesucht</a>
+        <a href=./about.html>Über uns</a>
+    </menu>
```

Comment:
Hi! Nice to meet you!😄 I'm Estefanía and I'll be reviewing your code today. 🔍 Let's see how we can leave it as functional and shiny as possible. 😀 Nice indentation here!